server_http.hpp

```diff
@@ -146,6 +146,15 @@ namespace SimpleWeb {
         write(StatusCode::success_ok, std::string(), header);
       }
 
+      void write_more(const std::string &content) {
+        *this << content;
+      }
```

Comment:
The write methods are just convenient methods for creating responses. For other cases, using the response stream through the `<<` operator is probably better than having a `write_more(const std::string&)` function, since the `<<` operator works with several other parameters as well.

Reply:
I'll give that a try.

Reply:
That works just fine. I've deleted my fork & gone back to your master branch. My new response code for mjpeg streaming now looks like this -- feel free to use as a streaming example if you like:

    // mjpeg stream resource
    server.resource["^/stream"]["GET"] = [this](std::shared_ptr<SimpleWeb::Server<SimpleWeb::HTTP>::Response> response,
                                                std::shared_ptr<SimpleWeb::Server<SimpleWeb::HTTP>::Request> request) {
        BOOST_LOG_TRIVIAL(info) << "GET /stream";
        std::thread streamThread([this, response] {
            class MJPEG {
            public:
                static void send_image(RestServer* server, const std::shared_ptr<SimpleWeb::Server<SimpleWeb::HTTP>::Response> &response) {
                    std::unique_lock<std::mutex> lk(server->streamMutex);
                    server->streamCv.wait(lk);
                    int size = (int)server->vJPEG.size();
                    *response << "--mjpegstream\r\n"
                            << "Content-Type: image/jpeg\r\n"
                            << str(boost::format("Content-Length: %1%\r\n\r\n") % size);
                    response->write((const char*)server->vJPEG.data(), size);
                    lk.unlock();
                    *response << "\r\n";
                    response->send([server, response](const SimpleWeb::error_code &ec) {
                        if (!ec) {
                            send_image(server, response);
                        }
                        else {
                            // Done
                            if (server->active > 0) {
                                server->active--;
                            }
                            BOOST_LOG_TRIVIAL(info) << "GET /image active--: " << server->active;
                        }
                    });
                }
            };
            SimpleWeb::CaseInsensitiveMultimap headers;
            headers.emplace("Content-Type", "multipart/x-mixed-replace;boundary=mjpegstream");
            headers.emplace("Connection", "close");
            headers.emplace("Cache-Control", "no-cache, no-store, max-age=0, must-revalidate");
            headers.emplace("Content-Length", "-1");
            response->write(SimpleWeb::StatusCode::success_ok, headers);
            active++;
            BOOST_LOG_TRIVIAL(info) << "GET /image active++: " << active;
            MJPEG::send_image(this, response);
        });
        streamThread.detach();
    };

Reply:
Thank you, the example is much appreciated!